Config.uk

```diff
@@ -168,6 +168,14 @@ config CROSS_COMPILE
 		need to set this unless you want the configured build
 		directory to select the cross-compiler automatically.
 
+
+config LLVM_TARGET_ARCH
+	string "Custom cross-compiler LLVM target (optional)"
+	help
+		Same as running 'make LLVM_TARGET_ARCH=target' but stored for
+		default make runs in this build directory.  You don't
+		need to set this unless you want the configured build
+		directory to select the cross-compiler automatically.
```

Comment:
Maybe add another sentence that this is used when compiling with LLVM compiles, like clang or rustc.